python/py-pycryptodome/Portfile

```diff
@@ -14,15 +14,14 @@ long_description    PyCryptodome is a self-contained Python package of \
                     low-level cryptographic primitives. \
                     PyCryptodome is a fork of PyCrypto.
 
-python.versions     27 35 36
+python.versions     27 35 36 37 38
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
-master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  ebdb0edb86f048889f467ddabb9efe95b74254e6 \
-                    sha256  be84544eadc2bb71d4ace39e4984ed2990111f053f24267a07afb4b4e1e5428f \
-                    size    6494255
+checksums           rmd160 a04415452518532e99d44dae2df38f7492431a52 \
+                    sha256 f1add21b6d179179b3c177c33d18a2186a09cc0d3af41ff5ed3f377360b869f2 \
+                    size 15451558
```

Comment:
please keep the indentations the same as before. i.e. the size checksum should line up.